### PR TITLE
Add standard linter

### PR DESCRIPTION
**Pull request by ahmedelgabri**
First try to add [`standard`](http://standardjs.com/) for #301

### CODE REVIEW

File: ale_linters/javascript/standard.vim [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        let l:text = l:match[3]
+
+        " vcol is Needed to indicate that the column is a character.
+        call add(l:output, {
+        \   'bufnr': a:buffer,
+        \   'lnum': l:match[1] + 0,
+        \   'vcol': 0,
+        \   'col': l:match[2] + 0,
+        \   'text': l:text,
+        \   'type': l:type ==# 'Warning' ? 'W' : 'E',
```
Comment (w0rp) [inline ale_linters/javascript/standard.vim]: Based on the assignment above, it looks like the type is always `'E'` here.

File: Dockerfile [hunk 2]
```diff
@@ -14,7 +14,7 @@ RUN apk --update add $PACKAGES && \
 
 RUN pip install vim-vint==0.3.9
 
-RUN npm install -g eslint@3.7.1
+RUN npm install -g eslint@3.7.1 standard@8.6.0
```
Comment (w0rp) [inline Dockerfile]: I don't think this is needed. The tests don't actually execute `standard`.

File: ale_linters/javascript/standard.vim (hunk 1 above)
Comment (ahmedelgabri) [inline ale_linters/javascript/standard.vim]: You are right

File: Dockerfile (hunk 2 above)
Comment (ahmedelgabri) [inline Dockerfile]: Will remove it then

**Comment by ahmedelgabri** [comment]
@w0rp updated the PR

**Comment by w0rp** [comment]
Cool, I'm happy with this.

I'll include this in the version after the version I'm trying to finish now.

**Comment by ahmedelgabri** [comment]
Great, thanks @w0rp

**Pull request merged**